lib/state/ui/reducer.ts

```diff
@@ -7,6 +7,22 @@ import * as T from '../../types';
 
 const emptyList: unknown[] = [];
 
+const dialogs: A.Reducer<T.DialogType[]> = (state = [], action) => {
```

Comment:
if we clear up the `{dialog: T.DialogType}` in the `action-types` file then the TypeScript error here goes away too